typescript/infra/src/tx/govern-transaction-reader.ts

```diff
@@ -450,8 +451,8 @@ export class GovernTransactionReader {
   ): boolean {
     return (
       tx.to !== undefined &&
-      timelocks[chain] !== undefined &&
-      eqAddress(tx.to!, timelocks[chain]!)
+      this.timelocks[chain] !== undefined &&
+      eqAddress(tx.to!, this.timelocks[chain]!)
```

Comment:
it seems like the timelocks in getGovernanceTimelocks() don't include the replaced timelocks (owner.ts). is this okay?